### DISCUSSION TITLE
Auto add Calendar Task / Event in shared with user

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#17074] [How to] [answered]
We have created Calendar Task / Event and assigned to other user, now in this case since module is configured as Private in Module Access, user who has created task can not see the task. In this case he has to mandatorily add himself in Shared With User, then he can see the task. Many times user forgets to add himself in shared with user list and loose the control over task.

Is there any way by which user who has created task can be added in shared with user list on create mode.

Jeevan Pendhari

**Comment by rskrzypczak** [comment, marked as answer]
You can try to use these parameters: 
https://github.com/YetiForceCompany/YetiForceCRM/blob/7ad1868c73aba9eced188cf998bccb649536a42a/config/Security.php#L53-L57
Otherwise, you'd have to write a handler to do it.

**Reply by JEEVAN28** [reply]
public static $permittedModulesByCreatorField = ['Calendar'];

Above solved my problem. @rskrzypczak Thanks for the help.
